tools/coredumper/sof-coredump-to-gdb.sh

```diff
@@ -8,11 +8,15 @@ elf="${1}"
 dump="${2}"
 
 reader_name="sof-coredump-reader.py" # in case it is changed
-reader_output="$(${THIS_SCRIPT_DIR}/${reader_name} -vc -i ${dump} --stdout -l 4)"
+reader_output="$(${THIS_SCRIPT_DIR}/${reader_name} -vc -i ${dump} -l 4 \
+		-o coredump-reader-output.txt)"
 reader_result="$?" # if $reader_name script fails, running xt-gdb is pointless
 if [[ ${reader_result} -ne 0 ]] ; then
 	echo "${reader_name} failed!"
 	exit ${reader_result}
 else
-	xt-gdb "${elf}" < <(echo "${reader_output}")
+	(echo "${reader_output}")
+	echo "quit" >> coredump-reader-output.txt
+	xt-gdb -q "${elf}" --command=coredump-reader-output.txt
```

Comment:
Can we have gdb binary as a option as some users will use gdb and others xt-gdb.  This can be done in a subsequent PR. i.e. 
```
sof-coredump-to-gdb.sh GDB=xtensa-cnl-elf-gdb
```